launch.sh

```diff
@@ -31,12 +61,16 @@ initialization() {
   fi
 
   # Get sources
-  git clone -q https://github.com/seapath/ansible
+  git clone -q "https://${SEAPATH_BASE_REPO}/ansible"
   cd ansible
-  git fetch -q origin ${GITHUB_REF}
+  git fetch -q origin "${GITHUB_REF}"
   git checkout -q FETCH_HEAD
   echo "Pull request sources got succesfully"
 
+  # Get inventories
+  rm -rf inventories_private
```

Comment:
not useful.
A new directory is created every time the ci is launched and all sources are downloaded in it. Their can be no "inventories_private" dir at this step.